resources/assets/sass/styles/topic.scss

```diff
@@ -217,7 +217,7 @@ TOPIC DETAILS
   display: none;
 }
 
-.tag {
+.topic-tags span  {
```

Comment:
redefining of style name breaks tags displaying: 
![selection_055](https://cloud.githubusercontent.com/assets/9032200/18792508/8672ffce-81bf-11e6-9db8-89015d74cd98.png)